### ISSUE TITLE
Extend peer dependencies to Angular 8

### ISSUE THREAD

**Issue by alvaro-octal** [Stabzs/Angular2-Toaster#188]
**Issue description**

running `npm install` throws the following warning:

```
npm WARN angular2-toaster@7.0.0 requires a peer of @angular/common@^7.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN angular2-toaster@7.0.0 requires a peer of @angular/compiler@^7.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN angular2-toaster@7.0.0 requires a peer of @angular/core@^7.0.0 but none is installed. You must install peer dependencies yourself.
```
**Steps to reproduce and a minimal demo of the problem**

`npm install` with Angular 8

**Current behavior**

`npm install` shows a couple of warning due to peer dependencies.

**Expected/desired behavior**

No warnings should be shown

**Comment by preston** [comment]
Yes please!!!

**Comment by dessalines** [comment]
Same.

**Comment by maxailloud** [comment]
Did you forget to publish to npm or is it just taking a bit of time to be updated?

**Comment by Stabzs** [comment]
It has just been published under version 8.0.0.  Thanks everyone for your patience!

**Comment by maxailloud** [comment]
Perfect thank you.
